Resources/Private/BackendModule/BackendModule.fusion

```diff
@@ -0,0 +1,85 @@
+prototype(NEOSidekick.AiAssistant:BackendModule) < prototype(Neos.Fusion:Component) {
+    scope = null
+    getAction = null
+    updateAction = null
+
+    # internal api
+    personalWorkspace = ${Neos.Ui.Workspace.getPersonalWorkspace()}
+    allowedTargetWorkspaces = ${Neos.Ui.Workspace.getAllowedTargetWorkspaces()}
+
+    renderer = Neos.Fusion:Component {
+        scope = ${props.scope}
+        workspaces = ${Array.set(props.allowedTargetWorkspaces, props.personalWorkspace.name, props.personalWorkspace)}
+        workspaces.@process.json = ${Json.stringify(value)}
+        endpoints = Neos.Fusion:DataStructure {
+            overview = Sitegeist.FusionLinkPrototypes:ModuleUri {
+                module = 'ai-assistant'
+                action = 'index'
+                absolute = true
+            }
+            nodeTypeSchema = Sitegeist.FusionLinkPrototypes:ActionUri {
+                package = 'Neos.Neos'
+                controller = 'Backend\\Schema'
+                action = 'nodeTypeSchema'
+                absolute = true
+                arguments = Neos.Fusion:DataStructure {
+                    # TODO: dirty hack to not have to re-implement neos:backend.configurationCacheVersion VH
```

Comment:
Not really a TODO, right? So change the comment

Reply:
Ist copy-paste von der Neos.Ui. Würde ich ggf. drinnen lassen als Reminder bis in der UI eine bessere Lösung kommt.

Reply:
Dann bitte klar beschreiben, woher der Code copy-pasted is, fühlt sich sonst etwas wie ein Plagiat an. Und macht das verstehen viel schwieriger